schema/schema.go

```diff
@@ -145,7 +145,7 @@ func (ta *Table) AddColumn(name string, columnType string, collation string, ext
 
 	if extra == "auto_increment" {
 		ta.Columns[index].IsAuto = true
-	} else if extra == "VIRTUAL GENERATED" {
+	} else if extra == "VIRTUAL GENERATED" || extra == "STORED GENERATED" {
 		ta.Columns[index].IsVirtual = true
```

Comment:
the name is `IsVirtual`, so don't save true for "STORED GENERATED"

Reply:
They all belong to automatically generated columns. Currently, go-mysql does not save the extra infomation. Need to define another variable in column to save it.

Reply:
Wouldn't `IsStored` for `STORED GENERATED` be more consistent with `IsVirtual` for `VIRTUAL GENERATED`?